### ISSUE TITLE
Create instance form has no submit button

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#369]
>
This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.

**Comment by msecret** [comment]
Closing into one issue.
